usb-agent/main.cpp

```
@@ -0,0 +1,43 @@
+
+    QCommandLineParser parser;
+    parser.setApplicationDescription(QStringLiteral("Moonlight USB bridge agent"));
+    parser.addHelpOption();
+    QCommandLineOption socketOption({ QStringLiteral("s"), QStringLiteral("socket") },
+                                     QStringLiteral("local IPC socket name"),
+                                     QStringLiteral("name"));
+    QCommandLineOption tokenOption({ QStringLiteral("t"), QStringLiteral("token") },
+                                    QStringLiteral("one-session IPC bearer token"),
+                                    QStringLiteral("token"));
+    parser.addOption(socketOption);
+    parser.addOption(tokenOption);
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)